packages/relay/tests/lib/hapiService.spec.ts

```diff
@@ -18,29 +18,37 @@
  *
  */
 
+import pino from 'pino';
 import path from 'path';
-import { expect } from 'chai';
 import dotenv from 'dotenv';
-import pino from 'pino';
+import { expect } from 'chai';
+import EventEmitter from 'events';
 import { Registry } from 'prom-client';
-import HAPIService from '../../src/lib/services/hapiService/hapiService';
-import { SDKClient } from '../../src/lib/clients';
 import { Client } from '@hashgraph/sdk';
+import { SDKClient } from '../../src/lib/clients';
 import HbarLimit from '../../src/lib/hbarlimiter';
+import HAPIService from '../../src/lib/services/hapiService/hapiService';
+import { CacheService } from '../../src/lib/services/cacheService/cacheService';
+
 dotenv.config({ path: path.resolve(__dirname, '../test.env') });
 
 const registry = new Registry();
 const logger = pino();
 
 describe('HAPI Service', async function () {
   this.timeout(20000);
-  let hapiService: HAPIService | null;
   let hbarLimiter: HbarLimit;
+  let cacheService: CacheService;
+  let eventEmitter: EventEmitter;
+  let hapiService: HAPIService | null;
```

Comment:
I don’t see any point in setting this variable to null in the tests, please remove such declarations:
```suggestion
  let hapiService: HAPIService;
```

Reply:
True. Updated